packages/events_derive/tests/macro.rs

```diff
@@ -0,0 +1,65 @@
+use error_stack::Result;
+use serde::Deserialize;
+
+#[allow(dead_code)]
```

Comment:
Why?

Reply:
clippy ignores debug printing for the usage analysis, so both fields are flagged as not used